docs/api-guide/serializers.md

```diff
@@ -189,6 +189,12 @@ Your `validate_<field_name>` methods should return the validated value or raise
                 raise serializers.ValidationError("Blog post is not about Django")
             return value
 
+---
+
+**Note:** If your `<field_name>` is declared on your model with the parameter `required=False` then this validation step will not take place.
```

Comment:
That's definitely one way to go about it.

The other thing that is that the note should be generic and apply to serializers, not just serializers that are generated from models. Though I think that change is only a matter of swapping "on your model" with "on your serializer", as I don't remember `required` being a valid Django model field argument.